set.mm

```diff
@@ -466126,6 +466419,224 @@ C_ dom ( ( ( T ` { I , J } ) o. F ) \ _I ) ) $=
       XALJBUUBGKUUFWKVCWOWLWMXAUUIUUJWPWTKXQXECUUGDUUGWCWNWQWRWS $.
   $}
 
+  ${
+    $d D i $.  $d D w $.  $d I i $.  $d J i $.  $d M i $.  $d U i $.  $d W i $.
+    $d W w $.  $d i ph $.
+    cycpmco2.c $e |- M = ( toCyc ` D ) $.
+    cycpmco2.s $e |- S = ( SymGrp ` D ) $.
+    cycpmco2.d $e |- ( ph -> D e. V ) $.
+    cycpmco2.w $e |- ( ph -> W e. dom M ) $.
+    cycpmco2.i $e |- ( ph -> I e. ( D \ ran W ) ) $.
+    cycpmco2.j $e |- ( ph -> J e. ran W ) $.
+    cycpmco2.e $e |- E = ( ( `' W ` J ) + 1 ) $.
+    cycpmco2.1 $e |- U = ( W splice <. E , E , <" I "> >. ) $.
+    $( The composition of a cyclic permutation and a well chosen 2-cycle is
```

Comment:
Maybe it's better to be more precise, like you were in the pull request description: "The composition of a cyclic permutation and a transposition of one element in the cycle and one outside the cycle, results in a cyclic permutation with one more element in its orbit.

Also, this is a humongous proof! metamath-exe has some trouble loading the proof, let alone minimzing it, so maybe it's worth splitting it into smaller lemmata.

Reply:
I've updated the comment in the last commit.

Yes it's a long proof, I will try to split it up. It also took ages for me to minimize.
(one obvious split is along the conjunction of the final statement)